meta.json

```diff
@@ -12,7 +12,7 @@
     "build": {
         "build": "./build.sh", 
         "path": "dist/archive.tar.gz", 
-        "arch": ["darwin/arm64", "darwin/amd64", "linux/arm64", "linux/amd64"] 
+        "arch": ["linux/arm64", "linux/amd64"] 
       },
     "entrypoint": "dist/main"
```

Comment:
Is the entrypoint `dist/__main__` or `dist/main` ?  it seems like the the Makefile and the meta.json do different things, they should be consistent

Reply:
that's my bad, fixing this